calcite-qml/Calcite/qmldir

```diff
@@ -0,0 +1,54 @@
+# Copyright 2012-2021 Esri
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Calcite
```

Comment:
🔧  I feel we should consider calling it Esri.Calcite

Reply:
I had initially gone down this route - but it triggered some bugs in theme discovery on Qt's end. It seems Qt doesn't expect the themes to be namespace qualified.

I can look into it a bit more hopefully this isn't make-or-break?

Reply:
Not make or break, just seemed like an ideal name